packages/blocks/src/__internal__/rich-text/reference-node.ts

```diff
@@ -69,8 +69,12 @@ export class AffineReference extends WithDisposable(ShadowlessElement) {
       white-space: break-spaces;
     }
 
+    .affine-reference-title {
+      color: var(--affine-text-primary-color);
+    }
     .affine-reference-title::before {
       content: attr(data-title);
+      color: var(--affine-link-color);
```

Comment:
:before pesudo element is render the reference title, which is blue. Originally it's color is inherit from .affine-reference class, but it will affect character's color that ime insert. I want it to be consistent with text style. So to solve this problem, I change the css.
<img width="270" alt="CleanShot 2023-04-24 at 09 53 13@2x" src="https://user-images.githubusercontent.com/24316656/233881974-c161d6bf-4c3e-4cbf-ad48-44ff4fa139a6.png">

<img width="257" alt="CleanShot 2023-04-24 at 09 53 32@2x" src="https://user-images.githubusercontent.com/24316656/233881998-b68a9cfe-b25b-4802-9ba5-cbc21af6b4ec.png">